JustRideSDKWrapper/dummy.m

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright 2023 Masabi Ltd
+ */
+
+// Swift Package Manager needs at least one header to prevent a warning. See
```

Comment:
Please change the part of this comment to: "A Swift Package Manager target needs..."